### DISCUSSION TITLE
shortcut applet shows messed up descriptions

### DISCUSSION THREAD

**Discussion by Themwhiskers** [HyDE-Project/HyDE#41] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/ed0a12f2-09ff-4814-ac7b-92afaa331ac8)
how can i fix it? I use it quite a lot tbh

**Comment by kRHYME7** [comment, marked as answer]
for now properly rename your binds into this format 

`bindd = mod,key, description,dispatcher,args `

this way the parser can read the proper description used for labels.

When I wrote the keybinds hint script in hyprdots there's no description option that's why I tried to parse comments for description, killing performance and reliability. In this repo we should use bindd, for the native description. I will update the keybindings.conf later to adhere to this convention.
